frontend/src/app/app-routing.module.ts

```diff
@@ -53,6 +53,13 @@ export const routes: Routes = [
 				(m: typeof import('./libs/common-components-demo/common-components-demo.module')) => m.CommonComponentsDemoModule
 			),
 	},
+	{
+		path: 'eventcreation',
```

Comment:
Path is visible for user, so, can we do it more prettier?
Like **create-event**.